public/app/plugins/datasource/azuremonitor/components/TracesQueryEditor/Filter.tsx

```diff
@@ -248,7 +248,6 @@ const Filter = (
         width={25}
       />
       <ButtonSelect<string>
-        placeholder="Operator"
```

Comment:
@grafana/partner-datasources can you double check this is ok? by the looks of it, `ButtonSelect` renders a regular html `<button>` which doesn't accept a `placeholder` prop, so i don't think this is doing anything. but i'm not sure where to look to test this 😅

Reply:
Nice catch :+1: fwiw you can view this button in the Traces view in Azure Monitor
![image](https://github.com/grafana/grafana/assets/1048831/9d2ee67c-c919-4f7e-a2d8-89540ef65dc3)